src/dbg/_scriptapi_module.cpp

```diff
@@ -179,4 +179,57 @@ SCRIPT_EXPORT bool Script::Module::GetList(ListOf(ModuleInfo) list)
         modScriptList.push_back(scriptMod);
     });
     return BridgeList<ModuleInfo>::CopyData(list, modScriptList);
-}
\ No newline at end of file
+}
+
+SCRIPT_EXPORT bool Script::Module::GetExports(const ModuleInfo* mod, ListOf(ModuleExport) list)
+{
+    if(mod == nullptr)
+        return false;
+
+    MODINFO* modInfo = ModInfoFromAddr(mod->base);
+    if(modInfo == nullptr)
+        return false;
+
+    std::vector<ModuleExport> modExportList;
+    modExportList.reserve(modInfo->exports.size());
+
+    for(auto & modExport : modInfo->exports)
+    {
+        ModuleExport entry;
+        entry.ordinal = modExport.ordinal;
+        entry.rva = modExport.rva;
+        entry.va = modExport.rva + modInfo->base;
+        entry.forwarded = modExport.forwarded;
+        strcpy_s(entry.forwardName, modExport.forwardName.c_str());

```

Comment:
Should be `strncpy_s(entry.forwardName, mod_export.forwardName.c_str(), _TRUNCATE)` to not hard crash if the name is too long.